README.md

```diff
@@ -70,3 +70,16 @@ Host bootstrap
 ```
 
 Please note the SNC project is “as-is” on this Github repository. At this time, it is not an offically supported Red Hat solution.
+
+## Building podman bundles
+
+It uses Fedora CoreOS stable channel to create the bundle.
+
+### How to use?
+- Clone this repo `git clone https://github.com/code-ready/snc.git`
+- `cd <directory_to_cloned_repo>`
+- `./snc.sh`
```

Comment:
How does this generate the podman only bundles? I don't see any flag or environment variable to create podman/openshift bundles?

Reply:
This is only the very beginning for this, as stated in the PR description:

>  this will get the master branch code closer to the one in the podman branch.

With these changes in the master branch+similar changes in the podman branch, `createdisk-library.sh` will be identical between both branches except for the generation of the crc-metadata-info.json file (this is what the `crc-bundle-generator` are about, but not finished yet).